examples/aws-ecs-fargate/README.md

```diff
@@ -1,29 +1,43 @@
 # AWS ECS with Fargate
 
-The CloudFormation document in this folder will create the following resources as well as the required rules, groups, and policies to make them work.
+The CloudFormation document in this folder will create the following resources, along with the necessary rules, groups, and policies to make them work.
 
-The result is a publicly accessible hostname that will route requests to a 1Password Connect instance running in AWS Fargate.
+The result is a publicly accessible hostname (via HTTPS) that routes requests to a 1Password Connect instance running in AWS Fargate. You can use your own VPC and subnets, or optionally set up DNS records automatically with Route 53. 
 
-If you are looking to deploy into existing an AWS VPC or use an existing ECS cluster please consider this CloudFormation as a starting point and you can modify it using a text editor or using Amazon's CloudFormation Designer tool.
+To customize further, like deploying into an existing ECS cluster, use this CloudFormation as a starting point and tweak it with a text editor or Amazon's CloudFormation Designer tool.
 
 ## Networking Resources
 
-- A VPC
-- 2 Public subnets
-- An Internet Gateway
-- An ALB
+- A VPC (optional, if you don’t provide your own)
+- 2 Public subnets (optional, if you don’t provide your own)
+- An Internet Gateway (optional, if creating a VPC)
+- An ALB (Application Load Balancer) with HTTPS
+- An ACM certificate for HTTPS
+- Route 53 DNS records (optional, if you provide a hosted zone ID)
```

Comment:
`[non-blocking suggestion]`

Would it make more sense to order these by which ones are needed first, followed by the optional ones? Also, putting the one that is needed based on whether another one is created  just below it might make it easier to identify dependencies.